config_obs.yml

```diff
@@ -141,6 +145,56 @@ aggregate:  # ------------------------------------- Climatological Operations --
   climatological_std: # automatically passed to the function
     op: std
 
+performance:  # ------------------------------------- Performance --------------------------------------------------
+  # Defines which statistical metrics to compute using functions from xsdba.measures.
+  dask:
+    n_workers: 8
+    threads_per_worker: 5
+    memory_limit: "8GB"
+  input:
+    observation: # Parameters used to search exclusively for observation data in the catalog.
+      type: ['station-pr', 'station-tas']
+    reconstruction: # Parameters used to search exclusively for reconstruction data in the catalog.
+      type: ['reconstruction']
+
+  # Expected "measures" YAML dictionary structure:
+  #
+  #   measures:
+  #      MEASURE:
+  #         - variable: VARIABLE1
+  #           OTHER_ATTR: VALUE1
+  #           ...
+  #         - variable: VARIABLE2
+  #           OTHER_ATTR: VALUE2
+  #           ...
+  #         - ...
+  #
+  # MEASURE: Which measure we're computing                                   e.g., 'rmse', 'mae', etc.
+  # VARIABLE: Which variables we're computing it for                         e.g., 'tg_mean_annual', 'pr_mean_seasonal'
+  # OTHER_ATTR: Other attribute(s) which allow us to select each variable    e.g., 'frequency', 'source'
```

Comment:
1. We would not call this attr, but rather `column` or `criteria`. Attr is much more general term to mean metadata attached to the data (https://docs.xarray.dev/en/latest/user-guide/data-structures.html)

but anyway

2. I don't see how this is necessary. Frequency is specified by the name. If we want a specific source, it would be specified in input.